src/connector/src/common.rs

```diff
@@ -51,15 +52,17 @@ pub struct AwsPrivateLinkItem {
 }
 
 #[serde_as]
-#[derive(Debug, Clone, Serialize, Deserialize)]
+#[derive(Debug, Clone, Serialize, Deserialize, WithOptions)]
 pub struct KafkaCommon {
+    #[with_option(required)]
```

Comment:
Can we derive this information from `Option<>` or not? i.e. Optional arguments are not required, and vice versa.

Reply:
Could you provide an example for this?